client-gen/src/snapshots/sails_clientgen__generator__tests__basic_works.snap

```diff
@@ -6,31 +6,37 @@ expression: generate(program).unwrap()
 use parity_scale_codec::{Decode, Encode};
 #[allow(unused_imports)]
 use sails_rtl::collections::BTreeMap;
-use sails_rtl::{String, *};
-#[allow(unused_imports)]
-use sails_sender::{Call, CreateProgramCall, GStdSender};
+use sails_rtl::{
+    calls::RemotingAction, gstd::calls::Args, gstd::calls::Remoting, prelude::*, String,
+};
 pub trait Service {
```

Comment:
Names for trait and its impl should reflect the name of the used IDL otherwise I will always need to use type aliasing. We discussed it earlier.

Reply:
Makes sense, implemented generating service name from idl file name